libs/application/template-api-modules/src/lib/modules/templates/estate/utils/mappers.ts

```diff
@@ -40,6 +41,11 @@ const estateMemberMapper = (element: EstateMember) => {
       : info(element?.nationalId).age < 18
       ? { nationalId: '', name: '', phone: '', email: '' }
       : undefined,
+    foreignCitizenship:
```

Comment:
is there really not already a utility method in monorepo that can check this instead of having to stitch this logic on the fly ?

Reply:
This is redundant here, as it's always going to be set in the client library, and will be removed